src/main/java/org/ibp/api/java/impl/middleware/dataset/DatasetCSVExportServiceImpl.java

```diff
@@ -77,19 +72,17 @@ void transformEntryTypeValues(final Map<Integer, List<ObservationUnitRow>> obser
 			entryTypes.stream().collect(Collectors.toMap(Enumeration::getDescription, Enumeration::getName));
 
 		final List<ObservationUnitRow> allRows =
-			observationUnitRowMap.values().stream().flatMap(list -> list.stream()).collect(Collectors.toList());
+			observationUnitRowMap.values().stream().flatMap(Collection::stream).collect(Collectors.toList());
 		allRows.forEach(row -> {
 			final ObservationUnitData data = row.getVariables().get(TermId.ENTRY_TYPE.name());
 			data.setValue(entryTypeDescriptionNameMap.get(data.getValue()));
 		});
 	}
 
-	void addLocationIdValues(final Map<Integer, List<ObservationUnitRow>> observationUnitRowMap) {
-		// FIXME IBP-3048: With location id now a property of StudyInstance, the query below is likely not needed
-		final Map<Integer, String> instanceIdLocationIdMap = this.studyDataManager.getInstanceIdLocationIdMap(new ArrayList<>(observationUnitRowMap.keySet()));
```

Comment:
@gelli-leafnode - I think we can also delete StudyDataManager#getInstanceIdLocationIdMap method already since it's rendered unused anymore.

Reply:
Applied code cleanup for StudyDataManager, StudyDataManagerImpl, StudyDataManagerImplTest, GeolocationPropertyDao, GeolocationPropertyDaoTest.